index.html

```diff
@@ -1,225 +1,110 @@
-            <path fill-rule="evenodd" d="M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z" />
-        </svg>
-        <span class="sr-only">Next</span>
-    </a>
-  </div>
-  <div class="row">
-<!--
-      <div class="col-md-7">
-        <p id="GeoResults"><iframe width=100% height=230px src="https://www.youtube.com/embed/wH5TUwzlaWI"></iframe></p>
-      </div>
-      <div class="col-md-5">
--->
-          <p>OData (Open Data Protocol) is an <a href="https://www.oasis-open.org/news/pr/iso-iec-jtc-1-approves-oasis-odata-standard-for-open-data-exchange">ISO/IEC approved</a>, <a href="https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=odata">OASIS standard</a> that defines a set of best practices for building and consuming RESTful APIs. OData helps you focus on your business logic while building RESTful APIs without having to worry about the various approaches to define request and response headers, status codes, HTTP methods, URL conventions, media types, payload formats, query options, etc. OData also provides guidance for tracking changes, defining functions/actions for reusable procedures, and sending asynchronous/batch requests.</p>
-          <p>OData RESTful APIs are easy to consume. The OData metadata, a machine-readable description of the data model of the APIs, enables the creation of powerful generic client proxies and tools. </p>
-<!--      
-    </div> 
--->
-  </div>
-
-<!--
-  <div id="CustomerQuotation" class="carousel slide row" data-ride="carousel">
-      <ol class="carousel-indicators">
-        <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
-        <li data-target="#myCarousel" data-slide-to="1"></li>
-        <li data-target="#myCarousel" data-slide-to="2"></li>
-      </ol>
-
-      <div class="carousel-inner" role="listbox">
-        <div class="item active" style="background-color:orange; color:black; padding:50px;">
-                <h1>"Blah blah blah blah blah"</h1>
-                <h2 class="text-right">-- Author Name, author title, company</h2>
-        </div>
-
-        <div class="item" style="background-color:orange; color:black; padding:50px;">
-          <h1>"Blah blah blah blah blah"</h1>
-          <h2 class="text-right">-- Author Name, author title, company</h2>
-        </div>
-
-        <div class="item" style="background-color:orange; color:black; padding:50px;">
-          <h1>"Blah blah blah blah blah blah blah blah"</h1>
-          <h2 class="text-right">-- Author Name, author title, company</h2>
+<div class="container">
+    <div class="row" style="margin: 4rem 0 6rem 0">
+        <h3 style="text-align:center; padding:2rem;">Featured Tools</h3>
+        <div class="items-container centered" style="flex-wrap:wrap; align-content:space-between">
+            {% assign featuredTools = site.tools | where: "featured", "true" %}
```

Comment:
we currently don't have any tools marked "featured", so this section is blank.  If it's blank, we should hide it (looks bad to have "Featured Tools" with nothing.)